# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../mod_python/mod_python-3.3.1-buckets.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: fix build failure, thanks to Tomo Vuckovic (#465246)
Reference: https://bugzilla.redhat.com/show_bug.cgi?id=465246
Origin: Fedora Project
http://pkgs.fedoraproject.org/gitweb/?p=mod_python.git;a=blob_plain;f=mod_python-3.3.1-buckets.patch;hb=HEAD

--- mod_python-3.3.1/src/connobject.c.buckets
+++ mod_python-3.3.1/src/connobject.c
@@ -139,7 +139,7 @@ static PyObject * _conn_read(conn_rec *c
     bytes_read = 0;
 
     while ((bytes_read < len || len == 0) &&
-           !(b == APR_BRIGADE_SENTINEL(b) ||
+           !(b == APR_BRIGADE_SENTINEL(bb) ||
              APR_BUCKET_IS_EOS(b) || APR_BUCKET_IS_FLUSH(b))) {
 
         const char *data;
